Roslyn.sln

```diff
@@ -426,6 +426,8 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.VisualStudio.Lang
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "IdeBenchmarks", "src\Tools\IdeBenchmarks\IdeBenchmarks.csproj", "{FF38E9C9-7A25-44F0-B2C4-24C9BFD6A8F6}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "CompilersIOperationGenerator", "src\Tools\Source\CompilerGeneratorTools\Source\IOperationGenerator\CompilersIOperationGenerator.csproj", "{D55FB2BD-CC9E-454B-9654-94AF5D910BF7}"
```

Comment:
Not sure it this matters, but should this be part of the src\Tools\Source\CompilerGeneratorTools\CompilerTools.sln as well?

Reply:
I don't think anyone uses that sln, if they need to update the project and want to use that sln they can update at that time.

---
In reply to: [309433193](https://github.com/dotnet/roslyn/pull/37624#discussion_r309433193) [](ancestors = 309433193)